lumaviewpro.py

```diff
@@ -4021,9 +4021,16 @@ def load_settings(self, filename="./data/current.json"):
                 self.ids['sequenced_image_output_format_spinner'].text = settings['image_output_format']['sequenced']
                 self.select_sequenced_image_output_format()
 
-                self.ids['frame_width_id'].text = str(settings['frame']['width'])
-                self.ids['frame_height_id'].text = str(settings['frame']['height'])
-                lumaview.scope.set_frame_size(settings['frame']['width'], settings['frame']['height'])
+                # Set Frame Size
```

Comment:
I'm wondering if this approach will handle the case where:
1. User is using LVP and enabled binning=2 (let's say resolution drops to 950x950).
2. User closes LVP, camera remains with setting binning=2 and 950x950. I can't remember if the camera settings will reset or get cleared out on a new LVP start, may be this is already handled.
3. User re-opens LVP.
4. The `load_settings()` function tries to set the frame size back to 1900x1900, but since the camera still has binning=2, this will not work correctly.

Would you be able to validate if this is an issue or not?  It may already be handled by the pyloncamera.py initialization.

Reply:
From testing, it seems to be working correctly. I tested it a few times with multiple combinations to be safe.